openedx_lti_tool_plugin/tests/test_signals.py

```diff
@@ -172,3 +178,112 @@ def test_without_graded_resources(
         all_from_user_id_mock.assert_called_once_with(user_id=self.user.id, context_key=self.course_key)
         datetime_mock.now.assert_not_called()
         self.graded_resource.update_score.assert_not_called()
+
+
+@patch('openedx_lti_tool_plugin.signals.send_vertical_score_update')
+@patch('openedx_lti_tool_plugin.signals.send_problem_score_update')
+class TestUpdateUnitOrProblem(TestCase):
+    """Test update_unit_or_problem_score signal."""
+
+    def setUp(self):
+        """Test fixtures setup."""
+        self.weighted_earned = 1
+        self.weighted_possible = 1
+        self.user_id = 1
+        self.course_id = COURSE_ID
+        self.usage_id = USAGE_KEY
+
+    @patch('openedx_lti_tool_plugin.signals.LtiProfile')
+    @patch('openedx_lti_tool_plugin.signals.settings')
+    @patch('openedx_lti_tool_plugin.signals.getattr')
+    def test_update_problem_score(
```

Comment:
Check calls to: getattr(settings, 'OLTITP_ENABLE_LTI_TOOL', False) and LtiProfile.objects.filter(user__id=user_id)